src/api/getAllConversations.ts

```diff
@@ -0,0 +1,16 @@
+import { idbDatabase, CONVERSATION_STORE_NAME } from './idb';
+import type { ConversationHistory } from '../types';
+
+export async function getAllConversations(): Promise<ConversationHistory[]> {
```

Comment:
Wouldn't we want to keep conversation histories as an object here?

https://github.com/Kava-Labs/hard-ai/blob/646afb9c266270fcd1f6cd6e4573f1710453a396/src/types.ts#L33